pkgs/applications/audio/roomeqwizard/default.nix

```diff
@@ -0,0 +1,91 @@
+{ writeTextFile, jdk8, runCommand, fetchurl, lib, makeWrapper, writers
+, common-updater-scripts, perl, nixfmt, nix, curl, coreutils, makeDesktopItem }:
+
+let
+  pname = "roomeqwizard";
+  version = "5.20.4";
+
+  responseFile = writeTextFile {
+    name = "response.varfile";
+    text = ''
+      createDesktopLinkAction$Boolean=false
+      executeLauncherAction$Boolean=false
+      mem$Integer=1
+      opengl$Boolean=false
+      sys.adminRights$Boolean=false
+      sys.installationDir=INSTALLDIR
+      sys.languageId=en
+      sys.programGroupDisabled$Boolean=true
+        '';
+  };
+
+  updateScript = writers.writeBash "${pname}-update.sh" ''
```

Comment:
Please use nix-shell.

Reply:
Not sure what you mean with this.

Reply:
Instead of injecting PATH please do it like https://github.com/NixOS/nixpkgs/blob/2e435a34d5bf50cda3883bdb07d6dd03f899b54c/pkgs/applications/misc/obsidian/default.nix#L22-L29